### PR TITLE
[1LP][RFR] Initial cluster graph commit

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- Added graph automation for Cluster

{{pytest: cfme/tests/candu/test_cluster_graph.py -v --use-provider complete}}

### CODE REVIEW

**Comment by digitronik** [comment]
Before PRT run https://github.com/ManageIQ/integration_tests/pull/7473 need to merge. `compare_data` share commonly for all graph from `cfme.tests.candu.__init__.py`

File: cfme/tests/candu/test_cluster_graph.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    collection = appliance.collections.hosts
+    for test_host in provider.data["hosts"]:
+        if not test_host.get("test_fleece", False):
+            continue
+        return collection.instantiate(name=test_host.name, provider=provider)
+
+
+@pytest.mark.uncollectif(
+    lambda provider, interval: provider.one_of(RHEVMProvider) and interval == "Daily"
+)
```
Comment (quarckster) [inline cfme/tests/candu/test_cluster_graph.py]: **REQUIRED**
Please provider `reason` argument.

File: cfme/tests/candu/test_cluster_graph.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    except AttributeError as e:
+        logger.error(e)
+    assert graph.is_displayed
+
+    def refresh():
+        provider.browser.refresh()
+        view.options.interval.fill(interval)
+
+    # wait, some time graph take time to load
+    wait_for(lambda: len(graph.all_legends) > 0, delay=5, timeout=200, fail_func=refresh)
```
Comment (quarckster) [inline cfme/tests/candu/test_cluster_graph.py]: **OPTIONAL**
```python
wait_for(lambda: bool(graph.all_legends), delay=5, timeout=200, fail_func=refresh)
```

**Comment by psav** [comment]
Needs rebase

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by digitronik** [comment]
@psav rebase done

**Pull request merged**